VisualPinball.Unity/VisualPinball.Unity/Import/VpxImporter.cs

```diff
@@ -59,9 +59,12 @@ public void Import(string fileName, Table table)
 			_table = table;
 
 			var go = gameObject;
-			go.name = _table.Name;
+			
 			MakeSerializable(go, table);
 
+			// set the gameobject name; this needs to happen after MakeSerializable because the name is set there as well
+			go.name = "Table [" + _table.InfoName + "]";
+

```

Comment:
As discussed, as GameObject names are in sync with the data names, this will break the table script.

Reply:
You've marked as resolved, but it's still there?

Reply:
Pushed a fix.